### ISSUE TITLE
[Request] Delete game files

### ISSUE THREAD

**Issue by ashleyhull-versent** [OtherCrashOverride/go-play#53]
Some games are just bad games. I'd like to delete unwanted games.

**Comment by rocky1138** [comment]
What I've been doing is putting my entire roms repository onto the card in a "backup" folder and just copying over the 2 or 3 games I am currently playing to the real roms folder. This works much better also because I no longer have to sift through 45 bazillion games to get to the one I want and the games are all still there if I ever do want to switch them up.

**Comment by JasonB32** [comment]
Sift through all the games on a windows emulator then delete them there, then transfer good one's to SD Card. That's what I do.

**Comment by celentanos** [comment]
I find the option to delete the games also useful. This can be done, for example, in the menu as an additional point to 'Restart' and 'Play / Resume'. For example, I could not play many games in my childhood, so I do not know which ones are good and which are bad ones. On the PC, I do not want to try this.

**Comment by dayue1984** [comment]
I need this too, few basic file manage functions

**Comment by shar33f** [comment]
I'd love a really simple file manager that would allow me to simply move roms between folders. For now, I keep entire romsets in a separate folder and then move the ones that I want to play in the correct folder. I carry a small USB-C microsd card reader on a keychain and use that connected to my phone to move roms when needed. Would be great to have a built in file manager right on the odroid go to handle this without having to swap out the microsd card.

**Comment by OtherCrashOverride** [comment]
I agree there is a need for file management.  However, I think it should be a stand alone application rather than part of the menu.  This would eliminate the need for other applications to also implement file management.

**Comment by DavidKnight247** [comment]
I have found Midnight Commander to be a nice file manager for consoles in
the past. Perhaps this could be ported over.
https://midnight-commander.org

**Comment by OtherCrashOverride** [comment]
That was actually my first thought too.  However, not having a keyboard is an obstacle to implementing existing file managers.

**Comment by shar33f** [comment]
I was just thinking a very simple UI to move or delete files (not rename them), so no keyboard needed. There is a simple file manager included on my GCW Zero that does basic move / delete using just the dpad and buttons, I can't remember the name but I'll check when I'm back home.

**Comment by asiag0** [comment]
@shar33f It's 'Dingux Commander'.  Here's the site for it: http://beyondds.free.fr/index.php?Dingoo-dinguxcommander

Not sure if it's opensource or not.

**Comment by shar33f** [comment]
Ah yes @asiag0 that's the one! The source code is on the same link you provided :)

**Comment by DatOdroidGuy** [comment]
Could also try a port of FBI from the 3DS since that lacks keyboard yet can delete, move and copy files just from A and B plus the D-Pad
